app/views/controllers/licenses/index.html.erb

```diff
@@ -0,0 +1,13 @@
+<%
+  @container = :wide
+  add_page_title(:index_license_header.t)
+  add_tab_set(license_index_tabs)
+
+  flash_error(@error) if @error && @objects.empty?
+%>
+
+<%= paginate_block(@pages) do %>
```

Comment:
Not a blocker, but I'd rather see this as a table than as what appears to be essentially a CSV.

Reply:
I'd also include the "display_name" in the table since that's more readable than the "form_name".